### PR TITLE
feat: update generated apis

### PR DESCRIPTION
**Pull request by jerome-quere**

### CODE REVIEW

File: api/instance/v1/instance_sdk.go [hunk 1]
```diff
@@ -2600,7 +2592,9 @@ type CreateSecurityGroupRequest struct {
 
 	Name string `json:"name,omitempty"`
 
-	OrganizationKey string `json:"organization_key,omitempty"`
+	Description string `json:"description,omitempty"`
+
+	Organization string `json:"organization,omitempty"`
```
Comment (kindermoumoute) [inline api/instance/v1/instance_sdk.go]: was this change intended?

Comment (DenBeke) [reply api/instance/v1/instance_sdk.go]: Apparently @jerome-quere changed it yesterday due to an error in the proto.

File: api/lb/v1/lb_sdk.go [hunk 2]
```diff
@@ -1542,45 +1542,45 @@ type UpdateBackendRequest struct {
 	// BackendID backend ID to update
```
Comment (kindermoumoute) [inline api/lb/v1/lb_sdk.go]: why this request remain public?

Comment (DenBeke) [reply api/lb/v1/lb_sdk.go]: Why would we make it private? 🤔
